src/index.rs

```diff
@@ -70,6 +70,8 @@ pub struct VariableIndexEntry {
     pub argument_type: ArgumentType,
     /// true if this variable is a compile-time-constant
     is_constant: bool,
+    // true if this variable is in a 'VAR_EXTERNAL' block
+    is_var_external: bool,
```

Comment:
```suggestion
    is_external: bool,
```

Reply:
I've considered naming it `is_external` too but then decided against it since it might be easy to mix-up with `{external}` linkage

Reply:
Yeah same here, I just thought for consistency sake (e.g. `is_constant`) we'd want it to be also called `is_external`, also related https://github.com/PLC-lang/rusty/pull/1324/files/0e46463c17a2ee1b0b055757beaf855c2e8586bf#r1791423510

But I'm fine either way, whichever one you prefer